src/pages/class/[classNumber]/manage.tsx

```diff
@@ -525,8 +541,10 @@ const Class: NextPage<{
         show={showAddContact}
         onClose={toggleShowAddContact}
         onSubmit={async (contact) => {
-          const { data: createdContact, error: contactCreationError } =
-            await createContact(contact);
+          const {
+            data: createdContact,
+            error: contactCreationError,
+          } = await createContact(supabase, contact);
 
           if (contactCreationError) {
             console.error(contactCreationError);
```

Comment:
Not sure when this was written but isn’t the error logged from inside the function anyway? If not then it should be.